### ISSUE TITLE
Finish order forms on the fund trading page

### ISSUE THREAD

**Issue by fubhy** [avantgardefinance/interface#181]
@jennazenk The order forms don't seem complete yet, what are the remaining pieces?

Here my list so far:

- [x] Add "cost" field (shows `quantity * price`), changing the cost field updates the quantity field.
- [x] Add styling for tabs (Orderbook / Kyber / Melon Engine)
- [ ] Add label for ask/bid somewhere (?) /cc @jennazenk
- [ ] Add switch (switch base/quote token) on Kyber form
- [x] Review and adjust style/positioning/etc. of all trading forms (ask Beatriz)
- [x] Add better validation for limit/market trading

**Comment by fubhy** [comment]
Also follow-up for https://github.com/avantgardefinance/interface/issues/168

**Comment by jennazenk** [comment]
Orderbook-based trading 

- [ ]  "Cost" field should be called "Total" and should not be greyed out. Total = quantity * price, but if total is changed by the user, this should be reflected in the "Quantity" field. Only the price field is greyed out in the case of market orders. 

- [ ] That being said (quantity and total not greyed out), the possible values must be controlled based on the quantity available at that price level. Quantity cannot exceed available quantity on that order. 

- [ ] Each field (quantity/price/total) must have an indication of what unit it is denominated in. 
Example: if we're on the MLN.WETH pair, quantity is expressed in MLN, price is in WETH per MLN and total is in WETH. See screenshot below. 

<img width="341" alt="Capture d’écran 2020-01-17 à 12 32 22" src="https://user-images.githubusercontent.com/17808182/72609672-d3e82100-3925-11ea-8a93-7721ffa1876b.png">


- [ ] Above the submit button, add a reformulation of the order in plain english: Market (or Limit) order: Buy (or Sell) XXX MLN at a price of XXX WETH per MLN. 

- [ ] It would be nice to add the option to select a "max available balance" button that fills the form as follows: for sell orders, max is shown below quantity, and clicking on max will fill the quantity field with your total available balance of MLN tokens. For a buy order, the max button is shown below the total field, and fills with your total available balance of WETH.

**Comment by jennazenk** [comment]
Cont'd in #273
